### ISSUE TITLE
Shorten log string about redacted sensitive info

### ISSUE THREAD

**Issue by dselyuzhitskiy** [ory/hydra#2377]
**Is your feature request related to a problem? Please describe.**

For common request, that have cookie value and query i see twice such message 

> Value is sensitive and has been redacted. To see the value set config key "log.leak_sensitive_values = true" or environment variable "LOG_LEAK_SENSITIVE_VALUES=true".

I think it is so long and unuseful

**Describe the solution you'd like**

May be just write 'redacted'

**Comment by aeneasr** [comment]
Redacted is missing instructions how to remove the redacted value - which is why we added it in the first place because people asked about it. How can we serve both purposes?

**Comment by zepatrik** [comment]
Maybe we can have `redacted - see https://ory.sh/redacted` that redirects to a docs page?

**Comment by aeneasr** [comment]
Ah, nice idea - react does this with error codes which redirect to a docs page. We could probably replicate that infrastructure?

**Comment by zepatrik** [comment]
Absolutely, as @Bin-fluence would say. That's where I got the idea from :wink:
